books/RayTracingTheNextWeek.html

```diff
@@ -710,7 +710,8 @@
 
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-            virtual bool bounding_box(double t0, double t1, aabb& output_box) const = 0;
+            virtual bool bounding_box(double time_start, double time_end, aabb& output_box)
```

Comment:
...
`time_start` `time_end` business again.
It might not be too big of a problem, but these do represent independent values to camera `time_start` and `time_end`

Reply:
Maybe just go ahead and change the camera's values in main to `camera_time_start`, `camera_time_stop`, or maybe `shutter_time_start`, `shutter_time_stop`

Reply:
Which, oh, _jesus_.
This `time_start` and `time_stop` values can also be independent from the `time0` and `time1` values of `moving_sphere`. But are probably (and always in the text) the same values.
"Fun" fact: if the bounding box's `time_start` is -inf and `time_end if +inf, the bounding encompasses all of space.

Reply:
The bbox seems reasonably infinite in that latter case. No surprise there.

Reply:
I'm ok with `camera_time_{start,stop}`.

Reply:
I think our views are different though. You seem to consider `time_start` and `time_end` as a sort of global time for all things, like `world_start` or `world_stop`. I just consider it a context-sensitive name, like `duration` or `length` or `magnitude`.

Reply:
That's fair. I'm just not used to many contexts where objects don't have sync'd times